src/Sarif.Multitool.FunctionalTests/Rules/UriMustBeAbsoluteTests.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool.Rules
+{
+    public class UriMustBeAbsoluteTests : SkimmerTestsBase
+    {
+        [Fact(DisplayName = nameof(UriMustBeAbsolute_ReportsInvalidSarif))]
+        public void UriMustBeAbsolute_ReportsInvalidSarif()
+        {
+            Verify(new UriMustBeAbsolute(), "Invalid.sarif");
```

Comment:
>UriMustBeAbsolute [](start = 23, length = 17)

Files should be named UriMustBeAbsolute_Valid.sarif and UriMustBeAbsolute_Invalid.sarif, as we discussed. Put them all in one directory. Now you can write a test that works just against your rule name and it will locate the file and do the right thing. All TBD of course, don't block.